libraries/standard/mqtt/src/mqtt_state.c

```diff
@@ -154,6 +245,26 @@ static bool validateTransitionPublish( MQTTPublishState_t currentState,
 
             break;
 
+        /* Below cases are for validating the resends of publish when a session is
+         * reestablished. */
+        case MQTTPubAckPending:
+
+            /* When a session is reestablished outgoing QoS1 publishes in state
+             * #MQTTPubAckPending can be resend. This will result in a no
+             * transition. */
```

Comment:
comma after reestablished; resend -> resent
how about "This will..." -> "The state remains the same."

Reply:
Thanks